server/routes/usersRoutes.js

```diff
@@ -8,5 +8,7 @@ const usersRoute = express.Router();
 usersRoute.post('/auth/signup', UsersController.userSignUp);
 // User can signin
 usersRoute.post('/auth/signin', UsersController.userSignIn);
+// Change user to the Mentor
+usersRoute.patch('/user/:id', UsersController.userToMentor);
```

Comment:
You need to also ensure that only an admin can change a user to a mentor. You can use a middleware to do this.

Reply:
yeah I want to finish all the endpoints so that I can  add middleware and add authorization after

Reply:
Alright